### ISSUE TITLE
Info "Required" is missing, in Settings

### ISSUE THREAD

**Issue by samuelinniger** [opencast-ilias/OpenCast#134]
**Describe the bug**
The info "Required" is missing, in the settings tabs. 
This info was shown in previos plugin versions, e.g. 3.7. 

**To Reproduce**
1. Open the Settings tabs, go to the bottom of the page

**Expected behavior**
Info "*Required" is shown (in german: "*Erforderliche Angabe").
Screenshot (Plugin Version 3.7)
<img width="572" alt="grafik" src="https://user-images.githubusercontent.com/60101424/214627421-c9790e68-3f95-4671-9591-94db5929b955.png">

**Screenshots**
Currently, nothing is shown in Plugin Version 5.1.

<img width="987" alt="grafik" src="https://user-images.githubusercontent.com/60101424/214627845-38e9928a-050b-4d78-9456-8801e3f6921b.png">

**Environment (please complete the following information):**
    ILIAS version: v7.11 2022-06-29
    Plugin version: 5.1
    Opencast version: 12
    Browser: Firefox

**Comment by mic-men** [comment]
Yes indeed. I can confirm this bug and we need the feature as well.

**Comment by dagraf** [comment]
> Yes indeed. I can confirm this bug and we need the feature as well.

@mic-men Do I understand you correctly, that you need the feature that there is a *Erforderliche Angabe message? Or do you refer on the missing "Ich akzeptiere die Nutzungsbedingungen"? If the latter is the case, see in the config of the plugin, that the location for accepting the EULA was moved from 'while creating a series' to 'uploading an event' (see screenshot). This feature was implemented by Uni Köln. 

<img width="810" alt="Bildschirmfoto 2023-02-08 um 12 59 06" src="https://user-images.githubusercontent.com/15246287/217523991-e83c2daf-ebd4-4a81-baf8-0bdd555cb487.png">

**Comment by mic-men** [comment]
Yes, you are right. Thanks for the hint.

**Comment by chfsx** [comment]
The missing "* Required" below a form was an issue in ILIAS Core which has been fixed in ILIAS 7.19.
